.github/workflows/workflow.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Build
```

Comment:
If we are thinking of managing this as an org workflow, these could be renamed to be more specific to the type of workflow e.g. `molecule.yml` and `name: Molecule`